source/standard_scores.Rmd

```diff
@@ -0,0 +1,1523 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all,-language_info
+    split_at_heading: true
+    text_representation:
+      extension: .Rmd
+      format_name: rmarkdown
+      format_version: '1.2'
+      jupytext_version: 1.14.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```{r setup, include=FALSE}
+source("_common.R")
+```
+
+# Ranks, quantiles and Standard Scores
+
+Sometimes have a set of measures, in some particular units, and we want some
+way to see quickly how these measures compare to one another, and how they may
+compare to other measures, in different units.
+
+Ranks are one way of having an implicit comparison between values.[^ordering]
+Is the value large in terms of the other values (with high rank) — or is it
+small (low rank)?
+
+[^ordering]: To get ranks we have to be able to sort our values in some
+    reasonable way, from high to low or low to high.  This usually makes sense
+    for measured data, where the measurement gives a number, but may not make
+    sense for other data, such as names or labels.  For example, it may ore may
+    not be meaningful to rank names alphabetically.  As usual, this is a matter
+    of judgment.
+
+If We can derive quantile positions from ranks.  These are values from 0 through
+1 that are closer to 1 for high rank values, and closer to 0 for low rank
+values.  Each value in the data has a rank, and a corresponding quantile
+position.  We can also look at the *value* corresponding to each quantile
+position, and these are the *quantiles*. You will see what we mean later in the
```

Comment:
Is the distinction between quantile and quantile position a standard notion? I don't see it used elsewhere, but I haven't searched extensively.

Reply:
I suppose I was confused, because in my mind quantiles are always between 0 and 1. But perhaps a small tweak, as in "we'll call these quantile values" would be sufficient?

Reply:
Well the *quantile positions* are always between 0 and 1 - or whatever you want to call the input values to `np.quantile(data, input_values)`.   Maybe suggest an edit?